<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Loder</class>
 <widget class="QMainWindow" name="Loder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>负载均衡器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>661</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_32">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_27">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>基本信息</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_26">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="0" column="1">
                 <widget class="QLineEdit" name="lineEdit_filePath"/>
                </item>
                <item row="0" column="2">
                 <widget class="QPushButton" name="Button_OpenFilePath">
                  <property name="text">
                   <string>打开</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>配置文件路径</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="QPushButton" name="Button_Switch">
                  <property name="text">
                   <string>调试开关(关)</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_7">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="gridLayout">
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="lineEdit_ID"/>
                  </item>
                  <item row="0" column="0">
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>Load_Balance_ID</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="1" column="0">
                 <layout class="QGridLayout" name="gridLayout_6">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>支持服务器数目 </string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="lineEdit_ServerNum"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item row="0" column="1">
               <layout class="QGridLayout" name="gridLayout_5">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="gridLayout_2">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>Client_udp_port</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="lineEdit_clientPort"/>
                  </item>
                 </layout>
                </item>
                <item row="1" column="0">
                 <layout class="QGridLayout" name="gridLayout_3">
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="lineEdit_serverPort"/>
                  </item>
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>server_udp_port</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>服务器配置信息</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_25">
           <item row="0" column="0">
            <widget class="QTableView" name="tableView_Server"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>中转消息显示</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_24">
           <item row="0" column="0">
            <widget class="QTableView" name="tableView_Msg"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>日志显示</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_23">
           <item row="0" column="0">
            <widget class="QTableView" name="tableView_Log"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout_31">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>统计</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_30">
          <item row="0" column="0">
           <widget class="QSplitter" name="splitter">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QWidget" name="">
             <layout class="QGridLayout" name="gridLayout_28">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_18">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="gridLayout_13">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_6">
                    <property name="text">
                     <string>从客户端接收消息数目</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <layout class="QGridLayout" name="gridLayout_11">
                    <item row="0" column="0">
                     <widget class="QLabel" name="label_7">
                      <property name="text">
                       <string>正确</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QLineEdit" name="lineEdit_ClientRight">
                      <property name="text">
                       <string>0</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item row="2" column="0">
                   <layout class="QGridLayout" name="gridLayout_12">
                    <item row="0" column="0">
                     <widget class="QLabel" name="label_8">
                      <property name="text">
                       <string>错误</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QLineEdit" name="lineEdit_ClientError">
                      <property name="text">
                       <string>0</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
                <item row="1" column="0">
                 <layout class="QGridLayout" name="gridLayout_17">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_12">
                    <property name="text">
                     <string>总计</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="lineEdit_ClientSum">
                    <property name="text">
                     <string>0</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <layout class="QGridLayout" name="gridLayout_22">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_15">
                  <property name="text">
                   <string>向客户端发送消息数目</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLineEdit" name="lineEdit_ClientSendSum">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="">
             <layout class="QGridLayout" name="gridLayout_29">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_19">
                <item row="1" column="0">
                 <layout class="QGridLayout" name="gridLayout_20">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_13">
                    <property name="text">
                     <string>总计</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="lineEdit_ServerSum">
                    <property name="text">
                     <string>0</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="0" column="0">
                 <layout class="QGridLayout" name="gridLayout_14">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_9">
                    <property name="text">
                     <string>从服务器接收消息数目</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <layout class="QGridLayout" name="gridLayout_15">
                    <item row="0" column="0">
                     <widget class="QLabel" name="label_10">
                      <property name="text">
                       <string>正确</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QLineEdit" name="lineEdit_ServerRight">
                      <property name="text">
                       <string>0</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item row="2" column="0">
                   <layout class="QGridLayout" name="gridLayout_16">
                    <item row="0" column="0">
                     <widget class="QLabel" name="label_11">
                      <property name="text">
                       <string>错误</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QLineEdit" name="lineEdit_ServerError">
                      <property name="text">
                       <string>0</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <layout class="QGridLayout" name="gridLayout_21">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_14">
                  <property name="text">
                   <string>向服务器发送消息数目</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLineEdit" name="lineEdit_ServerSendSum">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
         <zorder>layoutWidget</zorder>
         <zorder>layoutWidget</zorder>
         <zorder>layoutWidget_3</zorder>
         <zorder>layoutWidget_4</zorder>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="Button_Log">
         <property name="text">
          <string>日志查看</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>678</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
